alpha/lib/model/category.php

```diff
@@ -565,13 +562,27 @@ public function reSetFullIds()
 			$this->setFullIds($this->getId());
 		}
 	}
-	
+
+	protected function addDeleteCategoryTreeKuserJob($categoryId, $deleteCategoryDirectMembersOnly = false, $fullIds)
+	{
+		$filter = new categoryKuserFilter();
+		$filter->setFullIdsStartsWith($fullIds);
+		$filter->set('_category_direct_members', $deleteCategoryDirectMembersOnly);
+
+		$c = new Criteria();
```

Comment:
This test is wrong, you may have category-kuser objects that associated with one of the category descendants